### DISCUSSION TITLE
What is often said to be Spain's national dish?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#488] [Galaxy Trivia] [answered]
*list main ingredients as emojis*

**Comment by mayankjaiswal1709** [comment]
Spain's national dish is often said to be Paella! Here are the main ingredients listed as emojis:

🍚 - Rice
🍤 - Shrimp
🐔 - Chicken
🥘 - Saffron (for seasoning and yellow color)
🍅 - Tomato
🌶️ - Bell pepper (traditionally green)
🧄 - Garlic
🧅 - Onion
🥕 - Carrots (optional)
🥘 - Olive oil (for cooking)

Please note that the ingredients can vary depending on the region and the cook's preferences, and different types of paella may have different ingredients.

**Comment by thuongtruong1009** [comment, marked as answer]
Spain has a rich culinary tradition, and many dishes are considered national dishes. Some of the most popular include:

Paella: a rice dish that originated in Valencia and can include seafood, chicken, rabbit, or even snails.

Gazpacho: a cold soup made from tomatoes, peppers, cucumbers, and bread, typically served during the summer.

Fabada Asturiana: a hearty bean stew that is a staple in Asturias, in the north of Spain.

Cocido Madrileño: a stew that is a specialty of Madrid and typically includes chickpeas, pork, and various cuts of meat.

Tostadas: a dish made from fried bread and topped with various ingredients such as tomatoes, olive oil, and garlic.

Tapas: small plate dishes that can include a wide variety of ingredients and flavors, such as olives, cheese, ham, seafood, and more.

**Comment by ParanoidUser** [comment]
@mayankjaiswal1709, @thuongtruong1009, nice try! Paella is 100% the most famous dish from Valencia. However, all the ingredients can be a bit expensive and take a long time to cook. Therefore, it is usually a festive dish.

Any other breakfast ideas?

**Comment by ParanoidUser** [comment]
@NetJox4, @TheVigor, @twist0R, @wfrsk, do you know? 🍽️

**Reply by oscarxdev** [reply]
Yeah!!
One of my favorite foods :)

**Comment by peterbaccaro** [comment]
**Spanish Omelette** (aka Tortilla Espanola) is celebrated as a national dish by Spaniards.

The main ingredients are:

:potato: - Potatoes
:onion: - Onions
:egg: - Eggs

**Comment by oscarxdev** [comment]
Pote Asturiano (aka Pote Asturiano)
Queso Manchego (Aka Manchego cheese)
Miga de Pan (aka Bread crumbs)

Main ingredients Paella:
Arroz = Rice
Pollo = Chicken
Gamba o Gambon =  Shrimp
